src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -67,16 +67,13 @@
   <PropertyGroup Condition=" '$(TargetFramework)' == 'uap10.0.19041' ">
     <DefineConstants>$(DefineConstants);UWP_19041;UWP_18362;UWP_16299;UWP_14393</DefineConstants>
   </PropertyGroup>
-  <ItemGroup Condition=" '$(Configuration)'=='Release' ">
-    <!-- Manage Xamarin.Forms version for distribution -->
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```

Comment:
Do we need this package for Debug as well?

Reply:
Not sure TBH, from my references they add all the sourcelink information globally, I'm not sure if it's just for the sake of the sample or if it's something required.